pkg/datapath/iptables/iptables.go

```diff
@@ -138,6 +138,7 @@ func getFeedRule(name, args string) []string {
 // at the moment.
 func skipPodToPodConntrack(ipv6 bool) bool {
 	return !ipv6 && option.Config.EnableIPv4 &&
+		option.Config.InstallNoConntrackIptRules &&
```

Comment:
Could we detect all of the other conditions below during startup and, if any is not met, disable `InstallNoConntrackIptRules` with an info message? That's how we currently treat other, similar options such as BPF host routing.

Once we have that, we only need to check for `InstallNoConntrackIptRules` here.